docs/sources/mimir/manage/mimir-runbooks/_index.md

```diff
@@ -639,6 +639,27 @@ How to **investigate**:
       - If you encounter any Compactor resource issues, add CPU/Memory as needed temporarily, then scale back later.
       - You can also optionally scale replicas and shards further to split the work up into even smaller pieces until the situation has recovered.
 
+### MimirCompactorHasRunOutOfDiskSpace
+
+This alert fires when the compactor has run out of disk space at least once.
+When this happens the compaction will fail and after some time the compactor will retry the failed compaction, but it is very likely that on each retry it will just hit the same disk space limit again and it won't be able to recover on its own.
+
+How to **investigate**:
+
+- Look at the disk space usage in the compactor's data volumes.
+- Look for an error with the string "no space left on device" to confirm that the compactor ran out of disk space.
+
+How to **fix** it:
+
+- The only long-term solution is to give the compactor more disk space, this can be achieved by doing one or multiple of:
+  - Horizontally scale the compactors
+  - Grow each compactor's disk
+- If the number of blocks that the compactor is failing to compact is not very significant and you want it to skip them in order to focus on more recent blocks instead, you can consider marking the affected blocks for no compaction:
+  ```
+  # in this example the object storage backend is GCS
```

Comment:
We discourage comments in the code blocks and prefer the explanation to be in the text preceding the block.

Reply:
removed in this commit, it's pretty obvious that this is GCS anyway with the `-backend gcs` so I don't think this really needs an explanation:

https://github.com/grafana/mimir/pull/8278/commits/8bb0ed1a03c6a42cdcacd45c5d446c1b1c1adb8d